Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM kbase/kb_jre
```

Comment:
Can we move this to deployment

Reply:
Not really - the standard practice is to put the Dockerfile in the top level of the code repo. That is the way it is in all of the other repos.

Reply:
meh

Reply:
Basically a Dockerfile is equivalent to a Makefile or a build.xml, if that helps explain things